src/GovUk.Education.ExploreEducationStatistics.Notifier/Functions/ReleaseNotifier.cs

```diff
@@ -0,0 +1,178 @@
+    }
+
+    [FunctionName("ReleaseNotifier")]
+    // ReSharper disable once UnusedMember.Global
+    public async Task ReleaseNotifierFunc(
+        [QueueTrigger(ReleaseNotificationQueue)]
+        ReleaseNotificationMessage msg,
+        ILogger logger,
+        ExecutionContext context)
+    {
+        logger.LogInformation("{FunctionName} triggered",
+            context.FunctionName);
+
+        var config = _configurationProvider.Get(context);
+
+        var baseUrl = config.GetValue<string>(BaseUrlName);
+        var webApplicationBaseUrl = config.GetValue<string>(WebApplicationBaseUrlName).AppendTrailingSlash();
+        var tokenSecretKey = config.GetValue<string>(TokenSecretKeyName);
+
+        var notifyApiKey = config.GetValue<string>(NotifyApiKeyName);
+        var notificationClient = _notificationClientProvider.Get(notifyApiKey);
+
+        var storageConnectionStr = config.GetValue<string>(StorageConnectionName);
+        var table = await _storageTableService.GetTable(storageConnectionStr, SubscriptionsTblName);
+
+        // Send emails to subscribers of publication
+        var newReleaseTemplateId = msg.Amendment
+            ? config.GetValue<string>(ReleaseAmendmentEmailTemplateIdName)
+            : config.GetValue<string>(ReleaseEmailTemplateIdName);
+
+        var releaseSubscriberQuery = new TableQuery<SubscriptionEntity>()
+            .Where(TableQuery.GenerateFilterCondition("PartitionKey", QueryComparisons.Equal,
+                msg.PublicationId.ToString()));
+        var releaseSubscriberEmails = await GetEmails(table, releaseSubscriberQuery, logger);
+
+        foreach (var email in releaseSubscriberEmails)
+        {
+            var unsubscribeToken =
+                _tokenService.GenerateToken(tokenSecretKey, email, DateTime.UtcNow.AddYears(1));
+            var values = new Dictionary<string, dynamic>
+            {
+                // Use values from the queue just in case the name or slug of the publication changes
+                { "publication_name", msg.PublicationName },
+                { "release_name", msg.ReleaseName },
+                {
+                    "release_link",
+                    $"{webApplicationBaseUrl}find-statistics/{msg.PublicationSlug}/{msg.ReleaseSlug}"
+                },
+                {
+                    // NOTE: update_note not used by original release email template
+                    "update_note", msg.UpdateNote
+                },
+                { "unsubscribe_link", $"{baseUrl}{msg.PublicationId}/unsubscribe/{unsubscribeToken}" },
+            };
+
+            _emailService.SendEmail(notificationClient, email, newReleaseTemplateId, values);
+        }
+        var sentEmails = releaseSubscriberEmails.ToHashSet();
+
+        // Send emails to subscribers of any associated superseded publication
+        var newReleaseSupersededSubscribersEmailTemplateId = msg.Amendment
```

Comment:
Just wondering why this is prefixed with `new`?  My mind is trying to process if it's a temporary name that's accidently been left in because it's a 'new' variable, or if it's because it's a 'new release' or because it's a 'new template id'. 😆

Reply:
It was for "new release" -  not sure if this was me originally or not - but I'll remove it either way :)